types/debug/debug-tests.ts

```diff
@@ -1,11 +1,11 @@
 
 
-import debug = require("debug");
+import { debug, IDebugger } from "debug"
```

Comment:
This doesn't seem correct. Since this is a callable entity, it isn't legal in ES2015 to import this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```

Could you elaborate why you think this is a correct change?

Reply:
I've tried 
```ts
import * as debug from 'debug'
```
and
```ts
import debug from 'debug'
```
but alwayas got error with tsc or rollup, before i found correct `import = require()` during editing this d.ts.
So i think it should be more common useage.
At first commit f55af70 , i only change the defulat export but ci failed, i don't know what's wrong so have to  unwrap the namespace to common export .

Reply:
maybe should export debug as default for 
```ts
import debug from 'debug'
```
and interface IDebugger usage
```ts
import debug, { IDebugger } from 'debug'
```
?